components/HeaderNav.vue

```diff
@@ -80,10 +80,10 @@ const headerLinks = [
     text: "Brand",
     navigateTo: '/',
   },
-  // {
-  //   text: "Creator",
-  //   navigateTo: '/creator',
-  // },
+  {
+    text: "Creator",
+    navigateTo: '/creator',
+  },
```

Comment:
This means that there is now one more item in our navbar, which challenges its responsiveness.

One alternative would be varying the gaps between links and button from this:
```vue
      <div class="flex nav-links hidden lg:block headerLink font-semibold">
        <a
          v-for="link in headerLinks"
          :key="link.scrollTo + `-link`"
          class="hover:text-primary ml-[24px] text-xl cursor-pointer"
          :class="{'text-secondary': isActive(link)}"
          @click="navigateTo(link.navigateTo)"
        >{{ link.text }}</a>
      </div>
      <div class="ml-[44px] hidden lg:block">
        <button
          class="border-secondary border-2 bg-secondary text-white text-xl border-secondary text-xl px-3 py-1.5 rounded-lg hover-button-blue"
          @click="navigateTo('/book-a-demo')"
        >
          Get Started
        </button>
      </div>
```

to this:
```vue
      <div class="flex nav-links hidden lg:block headerLink font-semibold">
        <a
          v-for="link in headerLinks"
          :key="link.scrollTo + `-link`"
          class="hover:text-primary ml-3 xl:ml-6 text-xl cursor-pointer"
          :class="{'text-secondary': isActive(link)}"
          @click="navigateTo(link.navigateTo)"
        >{{ link.text }}</a>
      </div>
      <div class="ml-6 xl:ml-11 hidden lg:block">
        <button
          class="border-secondary border-2 bg-secondary text-white text-xl border-secondary text-xl px-3 py-1.5 rounded-lg hover-button-blue"
          @click="navigateTo('/book-a-demo')"
        >
          Get Started
        </button>
      </div>
```

So, basically, just changing classes from `class="hover:text-primary ml-[24px] text-xl cursor-pointer"` to `class="hover:text-primary ml-3 xl:ml-6 text-xl cursor-pointer"` and `<div class="ml-[44px] hidden lg:block">` to `<div class="ml-6 xl:ml-11 hidden lg:block">`

<img width="1126" alt="Screenshot 2024-07-29 at 6 04 05 PM" src="https://github.com/user-attachments/assets/7a467790-326d-4169-8535-77b27a6f7b47">